packages/toolpad-components/src/Text.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import {
+  Skeleton,
+  Typography as MuiTypography,
+  TypographyProps as MuiTypographyProps,
+  Link as MuiLink,
+  LinkProps as MuiLinkProps,
+  styled,
+} from '@mui/material';
+import { createComponent, TOOLPAD_COMPONENT_MODE_PROPERTY } from '@mui/toolpad-core';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+type BaseProps = MuiLinkProps | MuiTypographyProps;
+interface TextProps extends Omit<BaseProps, 'children'> {
+  [TOOLPAD_COMPONENT_MODE_PROPERTY]: string;
```

Comment:
Why not just `mode`?

Reply:
Agreed. `[TOOLPAD_COMPONENT_MODE_PROPERTY]` was envisaged to become a property that would reflect "rendering modes" across components. We don't need it since "mode" is just a prop now.